client/task-list/tasks/payments/components/WCPaySuggestionCard.js

```diff
@@ -39,14 +37,11 @@ const TosPrompt = () =>
 		},
 	} );
 
-export const WCPayMethodCard = ( { suggestion } ) => {
-	const { description, id } = suggestion;
-	const paymentGateway = useSelect( ( select ) => {
-		return (
-			select( PAYMENT_GATEWAYS_STORE_NAME ).getPaymentGateway( id ) || {}
-		);
-	} );
-	const { enabled: isEnabled, needs_setup: needsSetup } = paymentGateway;
+export const WCPaySuggestionCard = ( {
```

Comment:
Optional: just curious if `Card` adds to this component name.  Would `WCPaySuggestion` be equally as effective and easier to read?  Just thinking out loud, not a requirement to merge.

Reply:
I think I'd tend to agree. Renamed ✅